hazelcast/src/test/java/com/hazelcast/jet/config/JobConfigTest.java

```diff
@@ -226,6 +231,52 @@ public void addCustomClasspaths() {
         );
     }
 
+    @Test
+    @SuppressWarnings("rawtypes")
+    public void when_mutatingLockedJobConfig_then_fail() {
+        JobConfig jobConfig = new JobConfig();
+
+        URL mockUrl;
+        try {
+            File file = File.createTempFile("jobConfig", "suffix");
+            mockUrl = file.toURI().toURL();
```

Comment:
```suggestion
            mockUrl = new URL("https://example.com");
```
would it make sense to use that, instead of creating a temporary file?

Reply:
Unfortunately no. To be honest that was exactly my first implementation, but some methods that uses mockUrl need the URL that is pointing to the file.